Gruntfile.js

```diff
@@ -383,6 +383,23 @@ module.exports = function (grunt) {
             src: ['**', '!hmda-pilot.zip']
           }
         ]
+      },
+      'codedeploy': {
```

Comment:
Also, not sure if you are able to do 2 stages of zipping, but you need to zip the source up and then wrap it for CodeDeploy to work properly (or change how we deploy the AngularJS app) http://docs.aws.amazon.com/codedeploy/latest/userguide/how-to-plan-revision.html